Intermittent error when testing "jupytext --execute" on Windows

From time to time some of the tests that run `jupytext --execute` fail on Windows. An instance when this occured is [this run](https://github.com/mwouts/jupytext/runs/579370683?check_suite_focus=true).

The corresponding error message is below:
```
------------------------------ Captured log call ------------------------------
WARNING  traitlets:execute.py:599 Timeout waiting for IOPub output
________________________________ test_execute _________________________________

tmpdir = local('C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-runneradmin\\pytest-0\\test_execute0')

    @requires_nbconvert
    def test_execute(tmpdir):
        tmp_ipynb = str(tmpdir.join('notebook.ipynb'))
        tmp_py = str(tmpdir.join('notebook.py'))
    
        with open(tmp_py, 'w') as fp:
            fp.write("""1 + 2
    """)
    
        jupytext(args=[tmp_py, '--to', 'ipynb', '--execute'])
    
        nb = read(tmp_ipynb)
        assert len(nb.cells) == 1
>       assert nb.cells[0].outputs[0]['data'] == {'text/plain': '3'}
E       IndexError: list index out of range
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,9 @@ Next release (2020-04-??)
 - Jupytext is tested in `pip` and `conda` environments, on Linux, Mac OS and Windows, using Github actions (#487)
 - Documented that the YAML header can be created with either `--set-kernel`, `--set-formats`, or both (#485)
 
+**Fixed**
+- Skip the `jupytext --execute` tests when the warning _Timeout waiting for IOPub output_ occurs, which is the case intermittently on Windows (#489)
+
 1.4.2 (2020-04-05)
 ------------------
 

--- a/tests/test_execute.py
+++ b/tests/test_execute.py
@@ -5,7 +5,7 @@ from jupytext.cli import jupytext
 
 
 @requires_nbconvert
-def test_pipe_nbconvert_execute(tmpdir):
+def test_pipe_nbconvert_execute(tmpdir, caplog):
     tmp_ipynb = str(tmpdir.join('notebook.ipynb'))
     tmp_py = str(tmpdir.join('notebook.py'))
 
@@ -15,6 +15,8 @@ def test_pipe_nbconvert_execute(tmpdir):
 
     jupytext(args=[tmp_py, '--to', 'ipynb', '--pipe-fmt', 'ipynb',
                    '--pipe', 'jupyter nbconvert --stdin --stdout --to notebook --execute'])
+    if "Timeout" in caplog.text:
+        pytest.skip(caplog.text)  # Issue 489
 
     nb = read(tmp_ipynb)
     assert len(nb.cells) == 1
@@ -22,7 +24,7 @@ def test_pipe_nbconvert_execute(tmpdir):
 
 
 @requires_nbconvert
-def test_pipe_nbconvert_execute_sync(tmpdir):
+def test_pipe_nbconvert_execute_sync(tmpdir, caplog):
     tmp_ipynb = str(tmpdir.join('notebook.ipynb'))
     tmp_py = str(tmpdir.join('notebook.py'))
 
@@ -32,6 +34,8 @@ def test_pipe_nbconvert_execute_sync(tmpdir):
 
     jupytext(args=[tmp_py, '--set-formats', 'py,ipynb', '--sync', '--pipe-fmt', 'ipynb',
                    '--pipe', 'jupyter nbconvert --stdin --stdout --to notebook --execute'])
+    if "Timeout" in caplog.text:
+        pytest.skip(caplog.text)  # Issue 489
 
     nb = read(tmp_ipynb)
     assert len(nb.cells) == 1
@@ -39,7 +43,7 @@ def test_pipe_nbconvert_execute_sync(tmpdir):
 
 
 @requires_nbconvert
-def test_execute(tmpdir):
+def test_execute(tmpdir, caplog):
     tmp_ipynb = str(tmpdir.join('notebook.ipynb'))
     tmp_py = str(tmpdir.join('notebook.py'))
 
@@ -48,6 +52,8 @@ def test_execute(tmpdir):
 """)
 
     jupytext(args=[tmp_py, '--to', 'ipynb', '--execute'])
+    if "Timeout" in caplog.text:
+        pytest.skip(caplog.text)  # Issue 489
 
     nb = read(tmp_ipynb)
     assert len(nb.cells) == 1
@@ -89,7 +95,7 @@ a + 1
 
 
 @requires_nbconvert
-def test_execute_sync(tmpdir):
+def test_execute_sync(tmpdir, caplog):
     tmp_ipynb = str(tmpdir.join('notebook.ipynb'))
     tmp_py = str(tmpdir.join('notebook.py'))
 
@@ -98,6 +104,8 @@ def test_execute_sync(tmpdir):
 """)
 
     jupytext(args=[tmp_py, '--set-formats', 'py,ipynb', '--sync', '--execute'])
+    if "Timeout" in caplog.text:
+        pytest.skip(caplog.text)  # Issue 489
 
     nb = read(tmp_ipynb)
     assert len(nb.cells) == 1
@@ -106,7 +114,7 @@ def test_execute_sync(tmpdir):
 
 @requires_nbconvert
 @requires_ir_kernel
-def test_execute_r(tmpdir):  # pragma: no cover
+def test_execute_r(tmpdir, caplog):  # pragma: no cover
     tmp_ipynb = str(tmpdir.join('notebook.ipynb'))
     tmp_md = str(tmpdir.join('notebook.md'))
 
@@ -117,6 +125,8 @@ def test_execute_r(tmpdir):  # pragma: no cover
 """)
 
     jupytext(args=[tmp_md, '--to', 'ipynb', '--execute'])
+    if "Timeout" in caplog.text:
+        pytest.skip(caplog.text)  # Issue 489
 
     nb = read(tmp_ipynb)
     assert len(nb.cells) == 1
@@ -124,7 +134,7 @@ def test_execute_r(tmpdir):  # pragma: no cover
 
 
 @requires_nbconvert
-def test_execute_in_subfolder(tmpdir):
+def test_execute_in_subfolder(tmpdir, caplog):
     tmpdir.mkdir('subfolder')
 
     tmp_csv = str(tmpdir.join('subfolder', 'inputs.csv'))
@@ -144,6 +154,8 @@ sum(ast.literal_eval(line) for line in text.splitlines())
 """)
 
     jupytext(args=[tmp_py, '--to', 'ipynb', '--execute'])
+    if "Timeout" in caplog.text:
+        pytest.skip(caplog.text)  # Issue 489
 
     nb = read(tmp_ipynb)
     assert len(nb.cells) == 3

